src/panel/app_panel.js

```diff
@@ -93,4 +97,81 @@ export default class AppPanel {
       this.minify();
     }
   }
+
+  async setPoi(poi, options = {}) {
+    this.activePoiId = poi.id;
+    this.panels.forEach(panel => {
+      if (panel.isPoiCompliant) {
+        panel.setPoi(poi, options);
+      } else if (!options.isFromList && !options.isFromFavorite) {
```

Comment:
`isFromList` has been renamed :)
```suggestion
      } else if (!options.isFromCategory && !options.isFromFavorite) {
```